### PR TITLE
Added license file and enhanced readme

### PR DESCRIPTION
**Pull request by dwightgunning**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ## Stream Twitter clone
 
-This is a simple twitter clone app built with [Stream](http://getstream.io)'s API. It shows you how you can use [GetStream.io](https://getstream.io/ "GetStream.io") to built a site similar to Twitter.
+This is a simple Twitter clone app built with [Stream](http://getstream.io)'s API. It shows you how you can use [GetStream.io](https://getstream.io/ "GetStream.io") to built a site similar to Twitter.
```
Comment (hannesvdvreken) [inline README.md]: > to built

to build?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+
+You can sign up for a Stream account at https://getstream.io/get_started.
```
Comment (hannesvdvreken) [inline README.md]: can make this a hyperlink like `[https://getstream.io/get_started](https://getstream.io/get_started)`?

**Comment by hannesvdvreken** [comment]
👍  looking good

**Comment by dwightgunning** [comment]
Fixed typo in README.me pointed out by @hannesvdvreken; took a moment to add some extra boilerplate info as well (since we're working on consistency across the different projects).

**Pull request merged**